nearcore/src/config.rs

```diff
@@ -471,6 +475,24 @@ impl Config {
             panic!("Failed to write a config file {}", err);
         }
     }
+
+    pub fn rpc_addr(&self) -> Option<&String> {
+        #[cfg(feature = "json_rpc")]
+        if let Some(rpc) = &self.rpc {
+            return Some(&rpc.addr);
+        }
+        None
+    }
+
+    #[allow(unused_variables)]
+    pub fn set_rpc_addr(&mut self, addr: String) {
+        #[cfg(feature = "json_rpc")]
+        {
+            // TODO(mina86): Switch to get_or_insert_default() once it’s stable.
+            // See <https://github.com/rust-lang/rust/issues/82901>
```

Comment:
If there is todo usually it is best to create an issue for it and put the issue number in the parentheses

Reply:
I hear you, though normally I find it too much hassle for this kind of minimal-impact aesthetic changes. I’ve removed the comment for the time being.